assets/scss/components/compat/woocommerce/_product.scss

```diff
@@ -155,9 +155,11 @@
 			}
 		}
 
-		tr:not(:last-child) {
+		tr{
+		  display: block;
+		  &:not(:last-child) {
 			margin-bottom: $spacing-xs;
-			display: block;
+		  }
```

Comment:
@cristian-ungureanu It seems that WooCommerce has changed the markup here. If you look above these lines, we style `td.label` which doesn't exist anymore (they've changed it to a `th` tag). I think we should refactor this smarter.

We can change the whole block of SCSS to something more efficient yielding 100 bytes less in size: 

```
	div.product form.cart .variations {
		td, th {
		  padding: 0;
		  display: block; 
		  width: 100%;
		}

		tr{
		  margin-bottom: $spacing-sm;
		  display: grid;
		}
	}
```